pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-matrix/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit, pkgconfig, pidgin, json_glib, glib, http-parser } :
+
+let
+  version = "2016-07-11";
+in
+stdenv.mkDerivation rec {
+  name = "purple-matrix-${version}";
```

Comment:
Could you change this to `purple-matrix-unstable-${version}` as this is no released version, please?